tests/periph_rtt/main.c

```diff
@@ -51,9 +52,12 @@ int main(void)
     puts("Initializing the RTT driver");
     rtt_init();
 
-    puts("Setting initial alarm");
-    last = TICKS_TO_WAIT;
-    rtt_set_alarm(TICKS_TO_WAIT, cb, 0);
+    uint32_t now = rtt_get_counter();
+    printf("RTT now: %" PRIu32 "\n", now);
+
+    last = now + TICKS_TO_WAIT;
```

Comment:
what about also masking the `last` value like it's done in the callback function ? Maybe the 'set alarm' part could be factorized.
I'll test on ST based platform tomorrow but otherwise it looks ok.

Reply:
Added masking as suggested.